Web/HTML Assignment-1/Question-1.html

```diff
@@ -0,0 +1,176 @@
+<!DocType html>
+<html>
+	<head>
+		<Title>
+			New Account
+		</Title>
+	</head>
+	<Body>
+		<table align = "center">
+			<tr >
+				<td colspan ="2"><b>Create Your Account:</b></td>
+			</tr>
+			<tr>
+				<td align="right">Email ID:</td>
+				<td><input type="email" placeholder="Enter your email" ></td>
```

Comment:
There should be name attribute for all form input elements that you are submitting with form.